### DISCUSSION TITLE
Feature Request: Fast Transition / Network Roaming / 802.11r

### DISCUSSION THREAD

**Discussion by mogwai** [RaspAP/raspap-webgui#793] [Ideas]
Is there any plans to support this so that several pis' can be dotted around the house?

**Comment by billz** [comment]
IIRC, the Cypress CYW43455 datasheet mentions support for 802.11r. However it isn't clear if this is also true of the WLAN driver (bcrmfmac). I haven't tested an 802r AP with the latest kernel. If you've had success, share your results and I'll try replicating it.

A related feature request for 802.11w (#484) led to some testing which showed that this was problematic with both the driver and client devices. 

See also this discussion re: multiple APs #786
